scrapy_poet/page_input_providers.py

```diff
@@ -1,12 +1,12 @@
 """The Injection Middleware needs a standard way to build the Page Inputs dependencies
 that the Page Objects uses to get external data (e.g. the HTML). That's why we
-have created a repository of ``PageObjectInputProviders``.
+have created a colletion of Page Object Input Providers.
 
-The current module implements a ``PageObjectInputProviders`` for
-:class:`web_poet.page_inputs.HttpResponse`, which is in charge of providing the response
-HTML from Scrapy. You could also implement different providers in order to
-acquire data from multiple external sources, for example,
-Splash or Auto Extract API.
+The current module implements a Page Input Providers for
+:class:`web_poet.HttpResponse <web_poet.page_inputs.http.HttpResponse>`, which
+is in charge of providing the response HTML from Scrapy. You could also implement
+different providers in order to acquire data from multiple external sources,
+for example, Splash or Auto Extract API.
```

Comment:
Unrelated, but I wonder if we should take the chance to:

```suggestion
for example, from scrapy-playwright or from an API for automatic extraction.
```